provision/api/logging.go

```diff
@@ -20,15 +20,15 @@ func NewLoggingMiddleware(svc provision.Service, logger log.Logger) provision.Se
 	return &loggingMiddleware{logger, svc}
 }
 
-func (lm *loggingMiddleware) Provision(externalID, externalKey string) (res provision.Result, err error) {
+func (lm *loggingMiddleware) Provision(token, externalID, externalKey string) (res provision.Result, err error) {
 	defer func(begin time.Time) {
-		message := fmt.Sprintf("Method add thing and connect to proxy: new thing ID: %s took %s to complete", res.Thing.ID, time.Since(begin))
+		message := fmt.Sprintf("Method add thing and connect to proxy: token: %s, new thing ID: %v took %s to complete", token, res.Things, time.Since(begin))
```

Comment:
This log is not consistent. It should be something like `Method provision for token %s and thing %s ....`

Reply:
things is array

Reply:
AFAIK we try to avoid capital letters in logs. You could simply use `things`. I think that Sprintf support arrays of strings